<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resistenciasven</class>
 <widget class="QDialog" name="Resistenciasven">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Col_Val">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Converitr</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ok_Res">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>106</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="KColorCombo" name="kColor1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>38</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="color" stdset="0">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="KColorCombo" name="kColor2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>61</width>
     <height>38</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="color" stdset="0">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="KColorCombo" name="kColor3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>61</width>
     <height>38</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="color" stdset="0">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="KColorCombo" name="kColor4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>61</width>
     <height>38</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="color" stdset="0">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="KColorCombo" name="kColor5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>61</width>
     <height>38</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true"/>
   </property>
   <property name="color" stdset="0">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValorC">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>171</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="B4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>115</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>4 Bandas</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="B5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>115</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>5 Bandas</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
